lms/static/js/edxnotes/models/note.js

```diff
@@ -1,6 +1,6 @@
 ;(function (define) {
 'use strict';
-define(['backbone', 'js/edxnotes/utils/utils', 'underscore.string'], function (Backbone, Utils) {
+define(['backbone', 'js/edxnotes/utils/utils', 'underscore.string'], function (Backbone, Utils, str) {
```

Comment:
Because of the AMD definition, we should use the returned variable instead of assuming that "str" will be added to the underscore object. In fact, underscore.string can be used without underscore.